src/Modal/Modal.jsx

```diff
@@ -82,39 +82,43 @@ class Modal extends Component {
                       </CloseButton>
                     </Col>
                   </Row>
+                </HeaderArea>
 
-                  <BodyArea id="modal-body-area">{children}</BodyArea>
+                <BodyArea>
+                  <div id="modal-body-area" className="overflow-container">
```

Comment:
Why do we need 2 layers here? (body area + sub div). 

You should just be able to remove the diplay: flex from Body Area and keep flex: 1 there